test/src/e2e_vm_tests/test_programs/should_pass/language/generic_type_inference/Forc.toml

```diff
@@ -2,7 +2,7 @@
 authors = ["Fuel Labs <contact@fuel.sh>"]
 entry = "main.sw"
 license = "Apache-2.0"
-name = "generic-type-inference"
+name = "generic_type_inference"
```

Comment:
I changed the name of this test to be consistent with the rest of the tests, but I didn't change the actual contents. Do you know why it would be failing? @mitchmindtree

Reply:
Ah, must've been the lock file it's passing now!